src/emailpro/information/emailpro-information.html

```diff
@@ -173,8 +173,8 @@ <h3 class="oui-tile__title oui-header_4" data-translate="emailpro_tab_INFORMATIO
                         <li class="oui-tile__item">
                             <div class="oui-tile__definition">
                                 <strong class="oui-tile__term d-block" data-translate="emailpro_tab_INFORMATIONS_webmail"></strong>
-                                <a data-ng-href="https://{{exchange.hostname}}" class="d-block word-break" target="_blank" title="{{ 'emailpro_dashboard_new_window' | translate }}">
-                                    <span data-ng-bind="'https://' + exchange.hostname"></span>
+                                <a data-ng-href="https://{{exchange.webUrl}}" class="d-block word-break" target="_blank" title="{{ 'emailpro_dashboard_new_window' | translate }}">
```

Comment:
```suggestion
                                <a data-ng-href="https://{{exchange.webUrl}}" class="d-block word-break" target="_blank" rel="noopener">
```
?

Reply:
Actually one the only case where we should make sure there is a `rel="noopener"`, good catch 😅